docs/howto/system_testing.md

```diff
@@ -118,6 +121,30 @@ data "aws_ami" "latest-amzn" {
 
 Notice the use of the `TEST_RUN_ID` variable. It contains a unique ID, which can help differentiate resources created in potential concurrent test runs.
 
+### Kubernetes service deployer
+
+The Kubernetes service deployer requires the `_dev/deploy/k8s` directory to be present. It can include additional `*.yaml` files to deploy
```

Comment:
I just noticed here that we are expecting `*.yaml` files under `_dev/deploy/k8s`. Sorry I missed it in the [spec PR](https://github.com/elastic/package-spec/pull/130/files#diff-16e6de8c0f8acc5369cc32adf906667fdf37e0e570af8ae5a43a142f31edf668R5). Any reason we don't expect `*.yml` files like we do everywhere else?

Reply:
BTW, this is not a blocker for this documentation PR as this documentation reflects the reality. 

But if there is no good reason to go with `*.yaml` here instead of `*.yml` like we do everywhere else, I would prefer that we switch to `*.yml` files under `_dev/deploy/k8s` too (starting with the spec, etc.)

Reply:
Yes, it's on purpose as in the Kubernetes universe they're handling `*.yaml` files (not `*.yml`).

Reply:
Gotcha, thanks!